src/instance.rs

```diff
@@ -69,8 +68,8 @@ where
 }
 
 impl<'a, T: PyTypeInfo> PyRef<'a, T> {
-    pub(crate) fn from_ref(r: &'a T) -> Self {
-        PyRef(r, PhantomData)
+    pub fn from_ref(reference: &'a T) -> Self {
```

Comment:
I'm not for this change.
PyRef exists because we cannot always get `*mut ffi::PyObject` from `&T whre T: PyTypeInfo`.
So this change allows `pyclass` to use `as_ptr` without copying it to Python heap